data.js

```diff
@@ -0,0 +1,51 @@
+const user = {
+  email: 'linsy.joyner@gmail.com',
+  username: 'linsydeanna',
+  password: 'supersecretpassword',
+  _id: '592c8029c1e6f9d97b6b146b',
+  trips: [
+    {
+      name: 'Europe 2017',
+      departureDate: '10/9/17',
+      returnDate: '10/16/17',
+      cities: [
+        {
+          city: 'Amsterdam',
+          country: 'Netherlands',
+          arrivalDate: '10/10/17',
+          departureDate: '10/13/17',
+          id: '1234'
+        }
+      ],
+      locations: [
+        {
+          locationName: 'Anne Frank House',
```

Comment:
`name` instead of `locationName`, since it's inside the array called `locations`.